src/sentry/utils/apidocs.py

```diff
@@ -260,14 +269,42 @@ def extract_endpoint_info(pattern, internal_endpoint):
         )
 
 
+def flatten(l):
+    return [item for sublist in l for item in sublist]
+
+
+def resolve_nested_routes(pattern, current_parents=None):
+    """
+    Returns a list of tuples. The first element in the tuple is a RegexURLPattern. The second element is
+    a string list of all the parents that url pattern has.
+    """
+    from sentry.utils.compat import map
```

Comment:
@manuzope You can't import this at the top of the file?

Reply:
https://github.com/getsentry/sentry/pull/16911/files#diff-0829f6c7386afaa934fe3d2d5b713a41R24 scared me away but I tried putting it there and it works so...going to try putting it there and see if Travis agrees.

Reply:
I can't imagine a utility import like this would have circular dependency issues